pebblo/app/pebblo-ui/src/constants/keywordMapping.js

```diff
@@ -0,0 +1,34 @@
+export const KEYWORD_MAPPING = {
+  "normal-text": "Normal Text",
+  "medical-advice": "Medical Advice",
+  "harmful-advice": "Harmful Advice",
+  "board-meeting": "Board Meeting",
+  "consulting-agreement": "Consulting Agreement",
+  "customer-list": "Customer List",
+  "distribution/partner-agreement": "Distribution/Partner Agreement",
+  "enterprise-agreement": "Enterprise Agreement",
+  "executive-severance-agreement": "Executive Severance Agreement",
+  "financial-report": "Financial Report",
+  "loan-and-security-agreement ": "Loan And Security Agreement",
+  "merger-agreement": "Merger Agreement",
+  nda: "NDA",
```

Comment:
Since "nda" is a single (no space) object key, the quotes do not appear on it. We only need quotes around object keys if there are multiple words separated by a space.